Presentation.Web/Controllers/API/V2/External/ItSystemUsages/ItSystemUsageV2Controller.cs

```diff
@@ -436,6 +437,62 @@ public IHttpActionResult DeleteSystemUsageRelation([NonEmptyGuid] Guid systemUsa
                 .Match(FromOperationError, () => StatusCode(HttpStatusCode.NoContent));
         }
 
+        [HttpPost]
+        [Route("{systemUsageUuid}/external-references")]
+        [SwaggerResponse(HttpStatusCode.Created)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
```

Comment:
missing "badrequest" response code and it's actually the first possible response code if you step into the function